### ISSUE TITLE
@vue/cli-plugins-e2e-cypress: vue add e2e-cypress not found

### ISSUE THREAD

**Issue by maoyuyang** [vuejs/vue-cli#5073]
### Version
4.1.1

### Reproduction link
[https://registry.yarnpkg.com/vue-cli-plugin-e2e-cypress](https://registry.yarnpkg.com/vue-cli-plugin-e2e-cypress)

### Environment info
```
Environment Info:

  System:
    OS: macOS High Sierra 10.13.6
    CPU: (4) x64 Intel(R) Core(TM) i5-7360U CPU @ 2.30GHz
  Binaries:
    Node: 12.5.0 - /usr/local/bin/node
    Yarn: 1.16.0 - /usr/local/bin/yarn
    npm: 6.9.0 - /usr/local/bin/npm
  Browsers:
    Chrome: 79.0.3945.117
    Firefox: 71.0
    Safari: 13.0.4
  npmPackages:
    @vue/babel-helper-vue-jsx-merge-props:  1.0.0 
    @vue/babel-plugin-transform-vue-jsx:  1.1.2 
    @vue/babel-preset-app:  4.1.1 
    @vue/babel-preset-jsx:  1.1.2 
    @vue/babel-sugar-functional-vue:  1.1.2 
    @vue/babel-sugar-inject-h:  1.1.2 
    @vue/babel-sugar-v-model:  1.1.2 
    @vue/babel-sugar-v-on:  1.1.2 
    @vue/cli-overlay:  4.1.1 
    @vue/cli-plugin-babel: ^4.0.0 => 4.1.1 
    @vue/cli-plugin-eslint: ^4.0.0 => 4.1.1 
    @vue/cli-plugin-router:  4.1.1 
    @vue/cli-plugin-unit-jest: ^4.1.0 => 4.1.1 
    @vue/cli-plugin-vuex:  4.1.1 
    @vue/cli-service: ^4.0.0 => 4.1.1 
    @vue/cli-shared-utils:  4.1.1 
    @vue/component-compiler-utils:  3.1.0 
    @vue/preload-webpack-plugin:  1.1.1 
    @vue/test-utils: 1.0.0-beta.29 => 1.0.0-beta.29 
    @vue/web-component-wrapper:  1.2.0 
    babel-helper-vue-jsx-merge-props:  2.0.3 
    eslint-plugin-vue: ^5.0.0 => 5.2.3 
    jest-serializer-vue:  2.0.2 
    vue: ^2.6.10 => 2.6.10 
    vue-eslint-parser:  5.0.0 
    vue-hot-reload-api:  2.3.4 
    vue-jest:  3.0.5 
    vue-loader:  15.7.2 
    vue-router: ^3.1.3 => 3.1.3 
    vue-style-loader:  4.1.2 
    vue-template-compiler: ^2.6.10 => 2.6.10 
    vue-template-es2015-compiler:  1.9.1 
    vuex: ^3.0.1 => 3.1.2 
  npmGlobalPackages:
    @vue/cli: 4.1.1
```

### Steps to reproduce
run `vue add e2e-cypress` in an existed project

### What is expected?
the plugin should be added

### What is actually happening?
error An unexpected error occurred: "https://registry.yarnpkg.com/vue-cli-plugin-e2e-cypress: Not found".

**Comment by sodatea** [comment]
What's the result of `vue --version` on the terminal?
I think this should have been fixed since v3.11.
Since you have both npm and yarn installed on the system, the global `vue` binary may have been pointed to a wrong version.

**Comment by maoyuyang** [comment]
I updated the vue-cli and it works,  thanks !
